src/libdredd/include/libdredd/mutation_replace_binary_operator.h

```diff
@@ -59,6 +61,12 @@ class MutationReplaceBinaryOperator : public Mutation {
   [[nodiscard]] bool IsValidReplacementOperator(
       clang::BinaryOperatorKind op) const;
 
+  std::string getExpr(clang::ASTContext& ast_context) const;
```

Comment:
Should be `GetExpr` to conform to the Google style.

Also I think this deserves a comment. Is it meant to return a string corresponding to the non-mutated expression (because that crops up a few times)?

Reply:
Changed.